lib/Dialect/ESI/runtime/cpp/tools/esiquery.cpp

```diff
@@ -36,12 +39,40 @@ int main(int argc, const char *argv[]) {
   if (argc > 3)
     cmd = argv[3];
 
-  std::unique_ptr<Accelerator> acc = Accelerator::connect(backend, conn);
-  const SysInfo &info = acc->sysInfo();
+  try {
+    std::unique_ptr<Accelerator> acc = registry::connect(backend, conn);
+    const auto &info = *acc->getService<services::SysInfo>();
+
+    if (cmd == "version")
+      std::cout << "ESI system version: " << info.esiVersion() << std::endl;
+    else if (cmd == "json_manifest")
```

Comment:
these command names should be stored in a centralized location.

Reply:
Why? I generally only break out the storage into a constexpr if the same const needs to be referred to in multiple places, which isn't the case here.